packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/TGPTResource.java

```diff
@@ -0,0 +1,44 @@
+package software.uncharted.terarium.hmiserver.resources;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+@Path("/api/tgpt")
+@Tag(name = "TGPT REST Endpoints")
+@Slf4j
+public class TGPTResource {
+	@Inject
+	ObjectMapper mapper;
+
+	@ConfigProperty(name = "tgpt.baseUrl", defaultValue = "/chatty/")
+	String baseUrl;
+
+	@ConfigProperty(name = "tgpt.appUrl", defaultValue = "http://localhost:8078/chatty/")
+	String appUrl;
+
+	@ConfigProperty(name = "tgpt.wsUrl", defaultValue = "ws://localhost:8078/chatty_ws/")
+	String wsUrl;
+
+	@ConfigProperty(name = "tgpt.token", defaultValue = "89f73481102c46c0bc13b2998f9a4fce")
```

Comment:
Should this be in the clear?

Reply:
This is only for local dev.  In production we will have a secret keypair.